### ISSUE TITLE
Error in inter_section (line 163): Unable to perform assignment because the left and right sides have a different number of elements

### ISSUE THREAD

**Issue by socksandlfipflops** [JinghaoLu/MIN1PIPE#61]
Hi,
I'm getting the following error:

> Unable to perform assignment because the left and right sides have a different number of elements.
> 
> Error in inter_section (line 163)
>         idbatch(i) = idtmp;
> 
> Error in frame_reg (line 107)
>         m = inter_section(m, sttn, se, pixs, scl, sigma_x, sigma_f, sigma_d, maskc);
> 
> Error in min1pipe (line 107)
>                     [m, corr_score, raw_score, scl, imaxy] = frame_reg(m, imaxy1, se, Fsi_new, pixs, scl, sigma_x,
>                     sigma_f, sigma_d);

For info, I'm loading 4 .mat files containing the sequential videos. Not sure what's going on here... Thanks for any help in advance!

**Comment by JinghaoLu** [comment]
Sorry for the late reply. If you still have interest, I can take a look at the data you have the error with. You can share the demo data with me through cloud (any way you prefer). Thanks!

**Comment by socksandlfipflops** [comment]
Hi,

Thanks for getting back in touch!

Absolutely, I'm happy to send you the data we are using! Are you ok with
WeTransfer, or would you prefer something else?

Cheers,
Rob

**Comment by JinghaoLu** [comment]
Yeah as long as I can download, whichever way you prefer. And also I would need the set of parameters you used.
